nvflare/private/fed/client/client_engine.py

```diff
@@ -286,6 +290,7 @@ def _shutdown_client(federated_client, admin_agent, touch_file):
             federated_client.process.terminate()
 
         admin_agent.shutdown()
+        security_close()
```

Comment:
should this be called in "client_train.py" ? 
Since this concept is relative to "security_init"

Reply:
Will address this in another PR